src/Libero/ruleset.xml

```diff
@@ -60,6 +61,16 @@
             <property name="spacesCountBeforeColon" value="1"/>
         </properties>
     </rule>
+    <rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration">
+        <properties>
+            <property name="allAnnotationsAreUseful" value="true"/>
```

Comment:
Seems quite fragile to maintain a list like https://github.com/doctrine/coding-standard/blob/0247a0d440e196d93611bd0f3c561ee12bd82789/lib/Doctrine/ruleset.xml#L302-L342. Prefer the blacklisting in #35.